### ISSUE TITLE
Cannot find color scheme 'hybrid'

### ISSUE THREAD

**Issue by noamelf** [rafi/vim-config#20]
On running `make install` got the following error:

```
[dein]   unite-mark(1 change)
remote/host: python3 host registered plugins ['deoplete']
remote/host: generated the manifest file in "/home/noam/.config/nvim/.init.vim-rplugin~"
[dein] Runtimepath updated: (2016/07/03 18:50:14)
Error detected while processing /home/noam/.config/nvim/config/theme.vim:
line   18:
E185: Cannot find color scheme 'hybrid'
Press ENTER or type command to continue
```

**Comment by ewhal** [comment]
I'm also getting this error now and I've tried a full reinstall and everything.

**Comment by rafi** [comment]
@ewhal can you run `:call dein#clear_state()` and `:call dein#recache_runtimepath()`, exit and run again?

**Comment by rafi** [comment]
@ewhal this should be fixed as of e19f2a82575e24378335e5c428a2bc668c387089
